packages/flutter/test/material/range_slider_test.dart

```diff
@@ -833,6 +833,123 @@ void main() {
     expect(values.end, closeTo(80, 0.01));
   });
 
+  testWidgets('Range Slider onChangeEnd and onChangeStart are called on an interaction initiated by tap', (WidgetTester tester) async {
+    RangeValues values = const RangeValues(30, 70);
+    RangeValues startValues;
+    RangeValues endValues;
+
+    await tester.pumpWidget(
+      Directionality(
+        textDirection: TextDirection.ltr,
+        child: StatefulBuilder(
+          builder: (BuildContext context, StateSetter setState) {
+            return MediaQuery(
+              data: MediaQueryData.fromWindow(window),
+              child: Material(
+                child: Center(
+                  child: RangeSlider(
+                    values: values,
+                    min: 0,
+                    max: 100,
+                    onChanged: (RangeValues newValues) {
+                      setState(() {
+                        values = newValues;
+                      });
+                    },
+                    onChangeStart: (RangeValues newValues) {
+                      startValues = newValues;
+                    },
+                    onChangeEnd: (RangeValues newValues) {
+                      endValues = newValues;
+                    },
+                  ),
+                ),
+              ),
+            );
+          },
+        ),
+      ),
+    );
+
+    // Get the bounds of the track by shifting by the the overlay radius.
```

Comment:
This comment "shifting by the..." doesn't seem to match the code.

Reply:
Updated here and everywhere else

Reply:
Thanks for the update; now I understand what all the 24s mean!